### PR TITLE
feat: <input> support inputmode

### PR DESCRIPTION
**Pull request by yuanyan**
Close https://github.com/openkraken/kraken/issues/2

### CODE REVIEW

**Comment by andycall** [comment]
还应该有测试

**Comment by yuanyan** [comment]
目前只能在手机模拟器里看到效果，所以测试用例无法体现

**Comment by yuanyan** [comment]
> 还应该有测试

可以补充了看下是否有报错

**Comment by andycall** [comment]
是不是应该用一个手机来跑集成测试

File: kraken/lib/src/dom/elements/input.dart [hunk 1]
```diff
@@ -671,7 +717,9 @@ class InputElement extends Element implements TextInputClient, TickerProvider {
   void _startCursorTimer() {
     _targetCursorVisibility = true;
     _cursorBlinkOpacityController!.value = 1.0;
-    _cursorTimer = Timer.periodic(_kCursorBlinkWaitForStart, _cursorWaitForStart);
+    if (_cursorTimer == null) {
+      _cursorTimer = Timer.periodic(_kCursorBlinkWaitForStart, _cursorWaitForStart);
```
Comment (wssgcg1213) [inline kraken/lib/src/dom/elements/input.dart:721]: Tips~ 这里有一个简化的二元操作符: `??=`
https://dart.dev/codelabs/dart-cheatsheet#null-aware-operators

**Review by wssgcg1213 — APPROVED** [review]

**Review by temper357 — APPROVED** [review]

**Review by wssgcg1213 — APPROVED** [review]

**Comment by wssgcg1213** [comment]
Looks like failed with CI.

**Review by andycall — APPROVED** [review]

**Comment by yuanyan** [comment]
> Looks like failed with CI.

FAIL: clip 003
    Message: Expected an screenshot is not equal with "snapshots/css/css-overflow/clip.ts.196bb9771" snapshot.

这个经常偶发失败

**Pull request merged**